### PR TITLE
Dev spencer white qjwg

### PR DESCRIPTION
**Pull request by swhitey**
adding ROAS dashboard to FB project

### CODE REVIEW

**Comment by swhitey** [comment]
screen shot 2019-01-08 at 12 53 00 pm

**Comment by swhitey** [comment]
Added Commit to update the label: for the period comparison explore

File: fb_campaign_metrics_roas.dashboard.lookml [hunk 1]
```diff
@@ -0,0 +1,963 @@
+- dashboard: fb_campaign_metrics__roas
```
Comment (AnnaSerova) [inline fb_campaign_metrics_roas.dashboard.lookml]: Could you change the name to fb_campaign_metrics_roas? (Just 1 underscore)

Comment (AnnaSerova) [inline fb_campaign_metrics_roas.dashboard.lookml]: Also it should be extending the base dashboard and inheriting filters from that :)

**Comment by swhitey** [comment]
AP-67 Add Period Comparison explore to the config and default it to hidden: no.

**Comment by swhitey** [comment]
made Annas adjustments, extended from base and removed filters

**Comment by AnnaSerova** [comment]
🎉 Looks great!

**Pull request merged**